bokeh/plotting/gmap.py

```diff
@@ -0,0 +1,192 @@
+        if 'plot_height' in kw and 'height' in kw:
+            raise ValueError("Figure called with both 'plot_height' and 'height' supplied, supply only one")
+        if 'height' in kw:
+            kw['plot_height'] = kw.pop('height')
+        if 'width' in kw:
+            kw['plot_width'] = kw.pop('width')
+
+        opts = GMapFigureOptions(kw)
+
+        title = kw.get("title", None)
+        if isinstance(title, string_types):
+            kw['title'] = Title(text=title)
+
+        super(GMap, self).__init__(api_key=google_api_key, map_options=map_options,
+                                   x_range=Range1d(), y_range=Range1d(), **kw)
+
+        xf = MercatorTickFormatter(dimension="lon")
+        xt = MercatorTicker(dimension="lon")
+        self.add_layout(LinearAxis(formatter=xf, ticker=xt), 'below')
+
+        yf = MercatorTickFormatter(dimension="lat")
+        yt = MercatorTicker(dimension="lat")
+        self.add_layout(LinearAxis(formatter=yf, ticker=yt), 'left')
+
+        tool_objs, tool_map = _process_tools_arg(self, opts.tools)
+        self.add_tools(*tool_objs)
+        _process_active_tools(self.toolbar, tool_map, opts.active_drag, opts.active_scroll, opts.active_tap)
+
+    annular_wedge = _glyph_function(glyphs.AnnularWedge)
+
+    annulus = _glyph_function(glyphs.Annulus)
+
+    arc = _glyph_function(glyphs.Arc)
+
+    asterisk = _glyph_function(markers.Asterisk)
+
+    bezier = _glyph_function(glyphs.Bezier)
+
+    circle = _glyph_function(markers.Circle)
+
+    circle_cross = _glyph_function(markers.CircleCross)
+
+    circle_x = _glyph_function(markers.CircleX)
+
+    cross = _glyph_function(markers.Cross)
+
+    diamond = _glyph_function(markers.Diamond)
+
+    diamond_cross = _glyph_function(markers.DiamondCross)
+
+    hbar = _glyph_function(glyphs.HBar)
+
+    ellipse = _glyph_function(glyphs.Ellipse)
+
+    image = _glyph_function(glyphs.Image)
+
+    image_rgba = _glyph_function(glyphs.ImageRGBA)
+
+    image_url = _glyph_function(glyphs.ImageURL)
+
+    inverted_triangle = _glyph_function(markers.InvertedTriangle)
+
+    line = _glyph_function(glyphs.Line)
+
+    multi_line = _glyph_function(glyphs.MultiLine)
+
+    oval = _glyph_function(glyphs.Oval)
+
+    patch = _glyph_function(glyphs.Patch)
+
+    patches = _glyph_function(glyphs.Patches)
+
+    quad = _glyph_function(glyphs.Quad)
+
+    quadratic = _glyph_function(glyphs.Quadratic)
+
+    ray = _glyph_function(glyphs.Ray)
+
+    rect = _glyph_function(glyphs.Rect)
+
+    segment = _glyph_function(glyphs.Segment)
+
+    square = _glyph_function(markers.Square)
+
+    square_cross = _glyph_function(markers.SquareCross)
+
+    square_x = _glyph_function(markers.SquareX)
+
+    text = _glyph_function(glyphs.Text)
+
+    triangle = _glyph_function(markers.Triangle)
+
+    vbar = _glyph_function(glyphs.VBar)
+
+    wedge = _glyph_function(glyphs.Wedge)
+
+    x = _glyph_function(markers.X)
```

Comment:
I don't disagree, but intend to do it later in a different PR. The mechanism needs a path to customizing the example code depending on where it is mixed in, which will take more work than I can allot right now.